packages/persist/lib/records.ts

```diff
@@ -133,6 +133,7 @@ export async function persistRecords({
             return acc;
         }, 0);
 
+        metrics.increment(metrics.Types.BILLED_RECORDS_COUNT, new Set(summary.billedKeys).size, { accountId });
```

Comment:
@bastienbeurier do we still need the records count/size and modified records count/size for billing and platform usage?
If not we could remove the one we don't need to lower our datadog bill

Reply:
Good point. 

Can we just keep:
- `Monthly modified records`
- `Records in (size in bytes)`

 and remove: 
- `Records out (size in bytes)`
- `Records in (count)`
- `Records out (count)`
- `Records Modified (size in bytes)`
- `Records Modified (count)`
- `Records total count (snapshot)`

Reply:
Happy to keep all the metrics except "Records total count (snapshot)" without the account_id dimension, that should only be 6 metrics that can be helpful to debug

Reply:
keeping the metrics as is for now. Let's revisit after DD next bill

Reply:
I can already tell you the bill is bad :D